frontend/src/components/common/Label.tsx

```diff
@@ -13,9 +12,17 @@ export const Label: React.FunctionComponent<ILabelProps & React.HTMLAttributes<H
   ...rest
 }) => {
   return (
-    <p {...rest} className={classNames('label', rest.className)}>
+    <StyledLabel {...rest} className={classNames('label', rest.className)}>
       {required && <span className="req">*</span>}
       {rest.children}
-    </p>
+    </StyledLabel>
   );
 };
+
+const StyledLabel = styled.p`
```

Comment:
Wonder if we should start separating these into their own files as discussed last week.

Reply:
Others are separated into their own files, but as discussed, doing that makes the most sense for large files. For small files moving styles to an external file increases the difficulty of parsing a component.